Remove use of udev_set_log_priority API its gone in systemd >= 218

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: chromium-40.0.2214.91/ui/events/ozone/device/udev/device_manager_udev.cc
===================================================================
--- chromium-40.0.2214.91.orig/ui/events/ozone/device/udev/device_manager_udev.cc
+++ chromium-40.0.2214.91/ui/events/ozone/device/udev/device_manager_udev.cc
@@ -33,29 +33,9 @@ enum {
   SYS_LOG_DEBUG = 7,
 };
 
-// Log handler for messages generated from libudev.
-void UdevLog(struct udev* udev,
-             int priority,
-             const char* file,
-             int line,
-             const char* fn,
-             const char* format,
-             va_list args) {
-  if (priority <= SYS_LOG_ERR)
-    LOG(ERROR) << "libudev: " << fn << ": " << base::StringPrintV(format, args);
-  else if (priority <= SYS_LOG_INFO)
-    VLOG(1) << "libudev: " << fn << ": " << base::StringPrintV(format, args);
-  else  // SYS_LOG_DEBUG
-    VLOG(2) << "libudev: " << fn << ": " << base::StringPrintV(format, args);
-}
-
 // Create libudev context.
 device::ScopedUdevPtr UdevCreate() {
   struct udev* udev = udev_new();
-  if (udev) {
-    udev_set_log_fn(udev, UdevLog);
-    udev_set_log_priority(udev, SYS_LOG_DEBUG);
-  }
   return device::ScopedUdevPtr(udev);
 }
 
